docs/jetstream/operations.md

```diff
@@ -84,6 +84,33 @@ Argo updates should be tested on a separate cluster before applying them to prod
 		* Change version number to most recent release
 * The GKE cluster itself is updated automatically by GCP
 
+## Tooling and Metric Versioning
+
+Jetstream uses the same tooling and metric versions for an experiment across its entire analysis duration. This prevents inconsistent results, for example, when changes are made to how mozanalysis computes results or new default metrics are added in metric-hub mid-experiment.
+
+### Keeping track of tooling versions
+
+When a new version of jetstream is released, for example after some library updates, a new Docker container gets pushed to the [Artifact Registry](https://console.cloud.google.com/artifacts?project=moz-fx-data-experiments). The container installs a specific version of each library and can be uniquely identified by a SHA256 hash. A timestamp indicating when the container was uploaded is also available.
+
+Every time Jetstream runs and writes computed results to BigQuery, it tags the result tables with a last updated timestamp. However, the enrollments table won't update on new Jetstream runs, giving us an anchor from which to identify a consistent Jetstream image hash. 
```

Comment:
Or maybe (since it applies to both of the new sections) it would be better to make a new section like "How to use the latest tooling and metric definitions"?